zebra-chain/src/transaction/serialize.rs

```diff
@@ -335,7 +364,67 @@ impl ZcashDeserialize for Transaction {
                 let inputs = Vec::zcash_deserialize(&mut reader)?;
```

Comment:
Please add comments to v5 deserialization like the v5 serialization comments:
https://github.com/oxarbitrage/zebra/pull/90/files#diff-ea762b004cd64770c3f4900c77d53620d0723dac1ab023443542de4f55e85398R192-R207